migrations/2022-09-07-114618_pm-reports/down.sql

```diff
@@ -0,0 +1 @@
+drop view pm_report_view;
```

Comment:
The table should be dropped, not the view. Make sure `diesel migration redo` works, because its pry failing now without the table drop.